app/controllers/auth_controller.rb

```diff
@@ -8,7 +8,10 @@ def sign_out_action
   end
 
   def sign_in_action
-    return error! "Missing parameters", 400 unless [:email, :password].none? { |x| params[x].blank? }
+    return error! 'Missing parameters', 400 unless %i(email password)
```

Comment:
I _hope_ it's smart enough to tell that it would violate another rule to flip it… but I don't know